ext/config.w32

```diff
@@ -67,7 +67,22 @@ if (PHP_CASSANDRA != "no" &&
             "Timeuuid.c " +
             "Uuid.c " +
             "UuidInterface.c " +
-            "Varint.c", "cassandra");
+            "Varint.c" +
```

Comment:
Recommend merging this before #48 due to differences that will likely cause a merge conflict; sweeping file changes in the PR for the Windows PECL build.